.github/workflows/release-notes.yaml

```
@@ -0,0 +1,76 @@
+  actions: write
+
+jobs:
+  release-notes:
+    runs-on: ubuntu-latest
+    
+    steps:
+      - name: ⤵️ Check out code from GitHub
+        uses: actions/checkout@v4.2.1
+
+      - name: Extract release notes and version
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2181:style:3:6: Check exit code directly with e.g. 'if ! mycmd;', not indirectly with $?

(shellcheck)